src/toolchain/export-web.md

```diff
@@ -445,4 +450,4 @@ to compile your extension as well, but newer `emcc` versions might still work re
 
 [^2]: `emcc` is the name of Emscripten's compiler.
 
-[^3]: The primary reason for this is it is necessary to compile with `-sSHARED_MEMORY` enabled. The shipped `std` does not, so building `std` is a requirement. Related info on about WASM support can be found [here](https://github.com/rust-lang/rust/issues/77839).
+[^3]: The primary reason for this is it is necessary to compile with `-pthread` and `-Ctarget-feature=+atomics` enabled for multi-threaded builds. The shipped `std` does not, and may also build with other flags we don't use, so building `std` is a requirement. Related info on about WASM support can be found [here](https://github.com/rust-lang/rust/issues/77839).
```

Comment:
Could you split this onto multiple lines?

Reply:
Sure thing